docs/docs/06-tools/03-debugging.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Debugging
+id: Debugging
+description: Learn how to debug your Wing application
+keywords: [debugging, debug, test, vscode]
+---
+
+## Overview
+
```

Comment:
Is there a way to add some test for this?

Even a white box test Hofi verifies that we are passing all the right arguments/env to child processes will go a long way to protect against regressions

Reply:
I'll add a very simple test